mypy_django_plugin/django/context.py

```diff
@@ -189,7 +189,7 @@ def get_field_set_type_from_model_type_info(info: Optional[TypeInfo], field_name
 
         model_info = helpers.lookup_class_typeinfo(api, model_cls)
         for field in model_cls._meta.get_fields():
-            if isinstance(field, Field):
+            if isinstance(field, Field) and hasattr(field, 'attname'):
                 field_name = field.attname
```

Comment:
Can we fallback to something else with `getattr`?

Reply:
A full solution probably looks more like the `GenericForeignKey` bit below and fully unpacking the generic relation. I did this quick hack to unblock what I was working on.